website/app/styles/markdown/other-elements.scss

```diff
@@ -1,60 +1,59 @@
-.doc-markdown-content {
-
-  // IMAGES
-
-  > img,
-  > p img,
-  > li img,
-  > th img,
-  > td img {
-    box-sizing: content-box;
-    max-width: 100%;
-    background-color: #fff; // TODO
-
-    &[align="right"] {
-      padding-left: 20px;
-    }
-
-    &[align="left"] {
-      padding-right: 20px;
-    }
-  }
+// LINKS
 
-  // CODE SNIPPETS
-  // TODO do we need this or in reality we will not use them directly?
+.doc-markdown-a {
+  color: teal;
+}
 
-  code { // for the typographic styles see "typography"
-    margin: 0;
-    padding: 4px 8px;
-    color: white; // TODO
-    background-color: black; // TODO
-    border-radius: 3px;
-  }
+// IMAGES
 
-  pre {
-    word-wrap: normal;
+.doc-markdown-img {
+  box-sizing: content-box;
+  max-width: 100%;
```

Comment:
Note: The image could potentially be wider than 100% with these styles together with the padding below. (Don't know if this is just something temporary for testing purposes or some sort of special styling effect you're working on though.)

Reply:
yes, you're right (I copied this code from the GithHub markdown styling as a baseline, and didn't notice that they applied `box-sizing: content-box;` to the `img` element.

In any case I have removed for now the styling in case of `align` (we'll figure out later if we need it).